spec/features/user_views_all_devTools_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+feature "views a list of all items" do
+  let!(:devtools) {FactoryGirl.create_list(:devtool, 5 )}
+  context "as a guest" do
+    before { visit devtools_path }
+    scenario "guest sees all the development tools" do
+      devtools.each do |devtool|
+        expect(page).to have_content(devtool.title)
+      end
+    end
+
+    # scenario "guest only has guest privilages" do
+    #  expect(page).to not_have_content('Add a dev tool')
+    #  expect(page).to not_have_content('delete button')
+    #  expect(page).to have_content('Sign Up Button')
+    # end
+  end
```

Comment:
when possible, we only want finished code. Either remove this last commented out test when you have user_auth or finish it